source/edge-server/monitor.txt

```diff
@@ -0,0 +1,232 @@
+.. _edge-server-monitor:
+
+===================
+Monitor Edge Server
+===================
+
+.. meta::
+   :description: Learn how to monitor Edge Server instances.
+
+.. facet::
+   :name: genre 
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can view metrics and logs for your Edge Server instances to get a general 
+health status, or information about usage, performance, and errors.
+
+You can monitor an instance:
+
+- In the Atlas UI
+- In a UI on the local hardware
+- Through Admin API calls
+
+You can also forward logs to a MongoDB collection or an external service.
+
+This page contains details about how to view metrics and logs for your 
+Edge Server instances, and what information is available.
+
+Before You Begin
+----------------
+
+.. tabs::
+
+   .. tab:: Atlas UI
+      :tabid: atlas-ui
+
+      You need the following to manage Edge Servers in the Atlas UI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+   
+   .. tab:: Local UI
+      :tabid: local-ui
+
+      The local UI is enabled by default, and only requires a running
+      Edge Server instance.
+
+      On an untrusted host, you can disable the local UI by running the
+      following ``edgectl`` command:
+
+      .. code-block:: console
+
+         edgectl config --app-id=<YOUR-APP-ID> --disable-local-ui
```

Comment:
I found that if you pass the flag without a value, it's equivalent to "true". So I thought it was easier to omit the value. But since you're asking this, I realize this probably isn't obvious - especially as we document these flags as bools - so I'll update throughout.

(Also, I realize you don't have to pass the app-id anymore with changes to `edgectl` so I need to update this anyway.)